src/main/java/querqy/elasticsearch/rewriter/SimpleCommonRulesRewriterFactory.java

```diff
@@ -10,20 +10,31 @@
 import querqy.rewrite.commonrules.model.BoostInstruction;
 import querqy.rewrite.commonrules.select.ExpressionCriteriaSelectionStrategyFactory;
 import querqy.rewrite.commonrules.select.SelectionStrategyFactory;
+import querqy.rewrite.lookup.preprocessing.LookupPreprocessorType;
 
 import java.io.IOException;
 import java.io.StringReader;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 
 public class SimpleCommonRulesRewriterFactory extends ESRewriterFactory {
 
-    private static final SelectionStrategyFactory DEFAULT_SELECTION_STRATEGY_FACTORY =
-            new ExpressionCriteriaSelectionStrategyFactory();
+    public static final String CONF_IGNORE_CASE = "ignoreCase";
+    public static final String CONF_ALLOW_BOOLEAN_INPUT = "allowBooleanInput";
+//    public static final String CONF_BOOST_METHOD = "boostMethod";
```

Comment:
Thanks for adding these constants. Could you please remove the constants that are commented out?